app.js

```diff
@@ -47,6 +47,12 @@ var login = require('./routes/login');
 var register = require('./routes/register');
 var search = require('./routes/search');
 
+app.use(function(req, res, next) {
+    res.header("Access-Control-Allow-Origin", "*");
+    res.header("Access-Control-Allow-Headers", "Origin, X-Requested-With, Content-Type, Accept");
+    next();
```

Comment:
Please send this change in a separate pull request.
I have created an issue for that. see here: #8